ion-schema/src/schema.rs

```diff
@@ -53,15 +53,29 @@ impl Schema {
         SchemaTypeIterator::new(Arc::clone(&self.types), self.types.get_imports())
     }
 
+    /// Returns the requested type, if imported in this schema;
+    /// otherwise returns None.
+    pub fn get_imported_type<A: AsRef<str>>(&self, name: A) -> Option<TypeDefinition> {
+        let type_id = self.types.get_imported_type_id_by_name(name.as_ref())?;
+        Some(TypeDefinition::new(*type_id, Arc::clone(&self.types)))
+    }
+
     /// Returns the requested type, if present in this schema or a a built in type;
     /// otherwise returns None.
-    pub fn get_type<A: AsRef<str>>(&self, name: A) -> Option<TypeDefinition> {
+    pub fn get_built_in_or_defined_type<A: AsRef<str>>(&self, name: A) -> Option<TypeDefinition> {
         let type_id = self
             .types
             .get_built_in_type_id_or_defined_type_id_by_name(name.as_ref())?;
         Some(TypeDefinition::new(*type_id, Arc::clone(&self.types)))
     }
 
+    /// Returns the requested type, if present in this schema or a a built in type or imported in the schema;
+    /// otherwise returns None.
+    pub fn get_type<A: AsRef<str>>(&self, name: A) -> Option<TypeDefinition> {
```

Comment:
I know that the library is still considered unstable, but it is not a great idea to re-use a function name to mean something different.

Perhaps keep the existing `get_type` as is, and call this function `get_local_or_imported_type()` (built-in types are implicitly imported).

Reply:
Makes sense. Will change this accordingly.